gst/nnstreamer/tensor_query/tensor_query_common.c

```diff
@@ -93,6 +93,13 @@ gst_tensor_query_socket_receive (GSocket * socket, GCancellable * cancellable,
   gsize read;
   GError *err = NULL;
   GstMapInfo map;
+  GstMemory *out_mem;
+  gboolean need_alloc;
+
+  if (!socket || !outbuf) {
```

Comment:
```socket``` would never be NULL and

```c
*outbuf = gst_buffer_new_and_alloc (0UL);
```
returns NULL according to https://github.com/genesi/gstreamer/blob/9d3066642e54c032200b919281f05427b1014527/gst/gstbuffer.c#L341-L356. Is it right? 

If you want to null-checking for ```outbuf```, do after line 340.

Reply:
It seems it returns buffer object without memory allocated but NULL. (https://developer.gnome.org/gstreamer/stable/gstreamer-GstBuffer.html#gst-buffer-new-allocate)
While `*outbuf = gst_buffer_new_and_alloc (0UL);` is valid, I removed unnecessary logging. Thanks